pkg/ibmvpc/examples/out/lint_out/PartialTCPRespond_Lint

```diff
@@ -1,47 +1,50 @@
 "Blocked TCP response" issues:
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-In the connection from test-vpc1-ky/db-endpoint-gateway-ky[10.240.30.6] to test-vpc1-ky/vsi1-ky[10.240.10.4] TCP src-ports: 1-99,201-65535; TCP src-ports: 100-200 dst-ports: 1-9,51-65535 response is blocked
-In the connection from test-vpc1-ky/vsi2-ky[10.240.20.4] to test-vpc1-ky/vsi1-ky[10.240.10.4] TCP src-ports: 1-99,201-65535; TCP src-ports: 100-200 dst-ports: 51-65535 response is blocked
-In the connection from test-vpc1-ky/vsi3a-ky[10.240.30.5] to test-vpc1-ky/vsi1-ky[10.240.10.4] TCP src-ports: 1-99,201-65535; TCP src-ports: 100-200 dst-ports: 1-9,51-65535 response is blocked
+In the connection from "test-vpc1-ky/db-endpoint-gateway-ky[10.240.30.6]" to "test-vpc1-ky/vsi1-ky[10.240.10.4]" TCP src-ports: 1-99,201-65535; TCP src-ports: 100-200 dst-ports: 1-9,51-65535 response is blocked
+In the connection from "test-vpc1-ky/vsi2-ky[10.240.20.4]" to "test-vpc1-ky/vsi1-ky[10.240.10.4]" TCP src-ports: 1-99,201-65535; TCP src-ports: 100-200 dst-ports: 51-65535 response is blocked
+In the connection from "test-vpc1-ky/vsi3a-ky[10.240.30.5]" to "test-vpc1-ky/vsi1-ky[10.240.10.4]" TCP src-ports: 1-99,201-65535; TCP src-ports: 100-200 dst-ports: 1-9,51-65535 response is blocked
 ... (2 more)
 
 ________________________________________________________________________________________________________________________________________________________________________________________________________
 
 "NACL for which there are no resources attached to" issues:
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-network ACL corrode-kilogram-cola-mandated of VPC test-vpc1-ky has no resources attached to it
+network acl "corrode-kilogram-cola-mandated" of VPC "test-vpc1-ky" has no resources attached to it
```

Comment:
Why are we changing "ACL" to "acl"? I think the acronym should stay capitalized (everywhere, not just here).

Reply:
sure. somehow `toLower` got into my code ;-)